src/Symfony/Component/Validator/Constraints/Some.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Exception\MissingOptionsException;
+
+/**
+ * @Annotation
+ *
+ * @author Marc Morera Merino <yuhu@mmoreram.com>
+ * @author Marc Morales Valldepérez <marcmorales83@gmail.com>
+ */
+class Some extends AbstractComposite
+{
+    /**
+     * @var string
+     *
+     * Message for notice Min Violation
+     */
+    public $minMessage = 'At least {{ limit }} element of this collection should pass validation.|At least {{ limit }} elements or more of this collection should pass validation.';
+
+    /**
+     * @var string
+     *
+     * Message for notice Max Violation
+     */
+    public $maxMessage = '{{ limit }} or less element of this collection should pass validation.';
+
+    /**
+     * @var string
+     *
+     * Message for notice Exactly Violation
+     */
+    public $exactlyMessage = 'Exactly {{ limit }} element of this collection should pass validation.|Exactly {{ limit }} elements of this collection should pass validation';
+
+    /**
+     * @var int
+     *
+     * Min number of Success expected
+     */
+    public $min;
+
+    /**
+     * @var int
+     *
+     * Max number of Success expected
+     */
+    public $max;
+
+    /**
+     * @var int
+     *
+     * Exactly number of Success expected
+     */
+    public $exactly;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct($options = null)
+    {
+        parent::__construct($options);
+
+        if ((isset($this->min) || isset($this->max)) && isset($this->exactly)) {
```

Comment:
Why `isset()`? IMO this reads weird for properties. I would check that they are not `null` instead.